src/core/__tests__/props-updating.test.tsx

```diff
@@ -777,4 +790,19 @@ describe('onXXXChange', () => {
 
     expect(() => fireOptionChange('text', '1')).toThrow();
   });
+
+  it('is not called if option is not subscribable', () => {
+    const onPropChange = jest.fn();
+    isSubscribable.mockImplementation(() => false);
+    mount(
+      <TestComponent
+        text="0"
+        onTextChange={onPropChange}
+      />,
+    );
+
+    expect(onPropChange).toHaveBeenCalledTimes(0);
```

Comment:
It looks like the test has two acts.
It will be better to separate it into two tests
> is not called on creation
>is not called on update

Reply:
Also, it can be in separate module
describe("onXXXChange")
    describe("subscribable options")
       beforeEach
       afterEach
       it("is not called on create")
       it("is not called on update")

Reply:
Refactored